src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -365,6 +382,9 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig, protected
       case e: GoogleJsonResponseException =>
         logger.error(s"Error occurred executing Google request for ${googleProject.string} / $context", e)
         throw CallToGoogleApiFailedException(googleProject, context, e.getStatusCode, e.getDetails.getMessage)
+      case illegalArgumentException: IllegalArgumentException =>
```

Comment:
When the access token lookup fails, it returns an `IllegalArgumentException`

Reply:
I've tested using `FCToken=garbage` as well as using a valid token that expired. Both returned an IllegalArgumentException. Looking at the response more closely it says `expected primitive class, but got: class com.google.api.client.googleapis.json.GoogleJsonError` so the illegal argument...isn't the key but rather the returned error?

Reply:
The GoogleJsonError underneath the IllegalArgumentException has a message of `key error` in both the garbage token example and the valid but expired token example

Reply:
That's weird - the Google call failed but the library returns an IllegalArgumentException?

Is the exception cause a `GoogleJsonResponseException`? Maybe this would work (and might be better in a generic method like `executeGoogleRequestAsync`):
 
```
case e: Exception if e.getCause.isInstanceOf[GoogleJsonResponseException] =>
  val googleEx = e.getCause.asInstanceOf[GoogleJsonResponseException]
  throw CallToGoogleApiFailedException(googleProject, context, googleEx.getStatusCode, googleEx.getDetails.getMessage)
```

Reply:
That still gives `expected primitive class, but got: class com.google.api.client.googleapis.json.GoogleJsonError` unfortunately :(